Don't build static JS lib to make sure DTrace probes are picked up.

--- js/src/build/moz.build.orig	2017-04-16 18:00:50 UTC
+++ js/src/build/moz.build
@@ -71,7 +71,3 @@ if CONFIG['OS_ARCH'] == 'SunOS':
     ]
 
 OS_LIBS += CONFIG['REALTIME_LIBS']
-
-NO_EXPAND_LIBS = True
-
-DIST_INSTALL = True
